src/_pytest/debugging.py

```diff
@@ -185,22 +187,25 @@ def do_continue(self, arg):
                 self._continued = True
                 return ret
 
+            do_continue.__doc__ = pdb_cls.do_continue.__doc__
+
             do_c = do_cont = do_continue
 
             def do_quit(self, arg):
-                """Raise Exit outcome when quit command is used in pdb.
```

Comment:
Why not keep this as is?

Reply:
Seems reasonable to copy the original docstring from `do_quit`, which is the purpose of the PR... turning into a comment makes sure we do not lose that information.

Reply:
Yeah, I changed this because it won't be the real docstring at runtime after `__doc__` is set.